git/git-lfs.rst

```diff
@@ -49,15 +49,12 @@ Configuring Git LFS
 Basic configuration
 -------------------
 
-Once, after you've installed Git LFS, run:
+Run this command to add a ``filter "lfs"`` section to :file:`~/.gitconfig`:
```

Comment:
I think it's still good to say that this only needs to be done once per user (home directory), but it does need to be done before using Git LFS.